com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/textureset/MaxRectsLayoutStrategy.java

```diff
@@ -607,34 +607,57 @@ private int contactPointScoreNode (int x, int y, int width, int height) {
             return score;
         }
 
-        private RectNode findPositionForNewNodeContactPoint (int width, int height, int rotatedWidth, int rotatedHeight, boolean rotate) {
+        private RectNode evaluateContactPoint(int width, int height, int rotatedWidth, int rotatedHeight, boolean rotate, Rect currRect) {
```

Comment:
This function does the same thing, but only for one item; the loop is moved outside to allow parallelization.